tests/cli/test_cli_parser.py

```diff
@@ -237,7 +237,7 @@ def test_hybrid_executor_get_cli_commands_with_error(
         assert celery_executor_command.name in commands
         assert ecs_executor_command.name not in commands
         assert (
-            "Failed to load CLI commands from executor: airflow.providers.amazon.aws.executors.ecs.ecs_executor.AwsEcsExecutor"
+            "Failed to load CLI commands from executor: ::airflow.providers.amazon.aws.executors.ecs.ecs_executor.AwsEcsExecutor"
```

Comment:
Is it worth special casing the error message when team id is not present/used?

It's not likely users will hit this, so maybe it doesn't matter

Reply:
Yeah I agree with your second sentiment, it's a pretty unlikely scenario and I think what they get here will be enough. If they did provide a team-name or alias then it will show up in the message as well.